docs/workloads/azure/data/etl_pipelines/etl_intro_data_azure.md

```diff
@@ -0,0 +1,45 @@
+---
+id: etl_intro_data_azure
+title: ETL Overview
+sidebar_label: ETL Overview
+hide_title: false
+hide_table_of_contents: false
+description: Introduction to ETL Pipelines
+keywords:
+  - ingest
+  - silver
+  - gold
+  - bronze
+  - databricks
+  - etl
+  - medallion
+---
+
+Stacks Data Platform showcases an array of integrated data processing pipelines, designed to demonstrate its full
+spectrum of capabilities. Our goal is to offer a comprehensive user journey that begins at the data source and travels
+through multiple transformations within the data lake layers, culminating in visualizations on graphical dashboards.
+
+## Data pipelines
+
+The naming convention for the data pipelines originates in Databricks' Medallion Architecture, which emphasises
+structured data transformation layers:
+
+* [Data Ingestion](ingest_data_azure.md): The initial stage of data collection.
+* [Silver Pipeline](silver_data_azure.md): Intermediate processing and transformation stage.
```

Comment:
Possibly more descriptive to say:

Conformance, cleansing and data quality processing stage.

Reply:
updated